wgsl/index.bs

```diff
@@ -1242,7 +1239,7 @@ of a variable in [=storage classes/workgroup=] storage.
 
 <div class='example wgsl global-scope' heading="Workgroup variables sized by overridable constants">
   <xmp>
-    [[override]] let blockSize = 16;
+    override blockSize = 16;
```

Comment:
Are we confident that "override" is the right word here? Asking as a non-native speaker :)
It's a verb, not a noun like I'd expect it to be. E.g. if we simply extrapolate from "variable" then this would be "overridable".
Even htough "let" is a verb too, it makes more sense than "override".

Reply:
Good question.

"Override" _can_ be a noun, just like "setting" or "default" can be a noun.

Example:  "That setting is an override that takes effect instead of the default".  :-)

Reply:
Sure, but in this case the object we are declaring isn't an "override" by itself.
It's a value that can be overriden.

Reply:
Does anybody have a better word?